### PR TITLE
Add Regular Show: Mordecai and Rigby in 8-Bit Land

### PR DESCRIPTION
**Pull request by lolnhatanh123**
Adds Regular Show: Mordecai and Rigby in 8-Bit Land.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+needs_shared_font = false
+
+[[releases]]
+title = "00040000000FD000"
+region = "eur"
+release_date = "2013-08-11"
+
+[[releases]]
+title = "00040000000EBD00"
+region = "Hoa Kỳ"
```
Comment (CodingKoopa) [inline games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat]: This isn't a valid region.

File: games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat [hunk 2]
```diff
@@ -0,0 +1,14 @@
+title = "Regular Show: Mordecai and Rigby in 8-Bit Land"
+description = "Mordecai and Rigby travel through the game, defeating enemies they encountered in their past adventures. Eventually, they reach an 8-bit version of Exit 9B, created by Garett Bobby Ferguson and his son. \n\nGPU: GeForce GTX 950\nCPU: Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz\nMemory: 8 GB RAM (7.9 GB RAM usable)\nCurrent resolution: 1360 x 768, 60Hz\nOperating system: Microsoft Windows 10 Pro\n7x native\n60fps-66fps"
```
Comment (Leo626) [inline games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat]: Please use the wikipedia description.

> Regular Show: Mordecai and Rigby in 8-Bit Land is an action-adventure, space shooter, top down shooter, platform, shoot 'em up video game based on the Cartoon Network series Regular Show. It was developed by WayForward Technologies and published by D3Publisher.

File: games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "00040000000FD000"
+region = "eur"
+release_date = "2013-08-11"
```
Comment (Leo626) [inline games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat]: Day and Month are reversed. Should be `2013-11-08`.

File: games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat (hunk 1 above)
Comment (Leo626) [inline games/regular-show-mordecai-and-rigby-in-8-bit-land/game.dat]: Looked up the title ID. It's suppose to be for `usa`.

**Comment by lolnhatanh123** [comment]
Thank you"  I will fix it

**Comment by CodingKoopa** [comment]
Please address the review comments, or this will be closed.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**